projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/use-boost-tooltip-copy.ts

```diff
@@ -0,0 +1,103 @@
+import { __ } from '@wordpress/i18n';
+import React, { useMemo } from 'react';
+import { useProduct } from '../../../hooks/use-product';
+import usePurchases from '../../../hooks/use-purchases';
+
+const JETPACK_BOOST_PRODUCTS = [
+	'jetpack_boost_bi_yearly',
+	'jetpack_boost_yearly',
+	'jetpack_boost_monthly',
+];
+
+/**
+ * Gets the tooltip copy based on the Boost letter grade and other factors.
+ *
+ * @param {object} props - Component props.
+ * @param {string} props.speedLetterGrade - The Boost speed letter grade.
+ * @returns {React.ReactElement | string} A translated JSX Element or string.
+ */
+export function useBoostTooltipCopy( { speedLetterGrade } ) {
+	const slug = 'boost';
+	const { purchases, isFetchingPurchases } = usePurchases();
```

Comment:
I just merged a PR to update the usePurchases hook to use useQuery, this will have to be updated slightly. Might want to rebase and convert it to the `data/purchases` hook instead 😄

Reply:
Cool, thanks Dylan!  I updated `usePurchases()` to use `data/purchases`. 👍